tests/flows/test_deindex.py

```diff
@@ -1001,7 +1002,7 @@ async def test_run_partial_updates_of_concepts_for_document_passages(
         vespa_search_adapter=local_vespa_search_adapter,
     )
 
-    assert document_remove_1_post.concept_counts is None
+    assert document_remove_1_post.concept_counts == concept_counts__document_remove
```

Comment:
From my understanding the document passage will have the concepts on it that were provided on line 915 of this module and thus they should be there at the end as we aren't actually updating the family index as part of the `run_partial_updates_of_concepts_for_document_passages` function?

Reply:
Ah, yes! I'll change it to read the updated objects from S3.